include/tilck/kernel/uefi.h

```diff
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+#pragma once
+
+extern ulong uefi_rt_addr;
```

Comment:
As mentioned in the previous review, please let's avoid exporting global variables everywhere. Encapsulation matters even in a low-level C projects like this one. Let's just export a `uefi_set_rt_pointer()` function that just sets that variable once, after asserting that's NULL (e.g. `ASSERT(!uefi_rt_addr)`. Also, that variable can now be made static inside the uefi.c translation unit.